VK_App/VK_App/ViewControllers/SecondBarVC/NewsTableViewController.swift

```diff
@@ -18,31 +17,70 @@ final class NewsTableViewController: UITableViewController {
 
     private enum NewsCellType: Int, CaseIterable {
         case header
-        case content
+        case text
+        case image
         case footer
     }
 
     // MARK: Private properties
 
-    private let networkService = NetworkService()
+    private let networkService = NetworkNewsService()
+    private var isLoading = false
+    private var nextFrom = Constants.emptyText
     private var news: [NewsFeed] = []
+    private var mostFreshDate: Double?
 
     // MARK: Life cycle
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        fetchNews()
+        setViewController()
+    }
+
+    // MARK: Public Methods
+
+    func didTappedShowMore(_ cell: NewsTextCell) {
+        tableView.beginUpdates()
+        cell.isExpanded.toggle()
+        tableView.endUpdates()
     }
 
     // MARK: Private Methods
 
+    @objc func refreshNews() {
```

Comment:
```suggestion
    @objc private func refreshNews() {
```
Приватный метод сделай или перенеси под другую Марку